lemminx-maven/pom.xml

```diff
@@ -87,6 +89,17 @@
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.22.2</version>
+				<configuration>
+					<systemPropertyVariables>
+						<maven.repo.local>${maven.repo.local}</maven.repo.local>
+						<org.eclipse.lemminx.maven.searcher.RemoteRepositoryIndexSearcher.disableCentralIndex>false</org.eclipse.lemminx.maven.searcher.RemoteRepositoryIndexSearcher.disableCentralIndex>

```

Comment:
For the disableCentralIndex, as this is also desired for any testing (not only Maven), we could consider building a TestSuite referencing all tests and setting this as part of the test suite.

Reply:
I'm not sure I fully understand - I was under the impression that disableCentralIndex was already affecting the entire test suite? I guess it'll be easier to understand once you show me the new patch :)